roiextractors/segmentationextractor.py

```diff
@@ -204,6 +221,27 @@ def get_image_size(self) -> ArrayType:
         """
         pass
 
+    def get_traces(self, roi_ids=None, start_frame=None, end_frame=None, name='Fluorescence'):
+        """
+        Return RoiResponseSeries
+        Returns
+        -------
+        traces: array_like
+            2-D array (ROI x timepoints)
+        """
+        if roi_ids is None:
+            roi_idx_ = range(self.get_num_rois())
+        else:
+            roi_idx = [np.where(np.array(i) == self.get_roi_ids())[0] for i in roi_ids]
+            ele = [i for i, j in enumerate(roi_idx) if j.size == 0]
+            roi_idx_ = [j[0] for i, j in enumerate(roi_idx) if i not in ele]
+        traces = self._roi_response_dict.get(name)
+        if len(traces.shape) == 0:
+            print(f'traces for {name} not found, enter one of {list(self._roi_response_dict.keys())}')
+            return None
```

Comment:
This should throw an error

Reply:
fixed error where it calls shape on `None`